scripts/create_gitlab_snapshot.py

```diff
@@ -29,29 +29,20 @@
 def main():
     require(config.terraform_component == 'gitlab',
             "Select the 'gitlab' component ('dev.gitlab' or 'prod.gitlab', for example).")
-    instance = gitlab_instance_info()
-    shutdown_instance(instance)
     volume = gitlab_volume_info()
+    instance = one(volume['Attachments'])
```

Comment:
Obviously, this shouldn't fail if there are no attachments. The phrase "if any" in my previous review made that clear.